script/proxy-server

```diff
@@ -15,7 +15,7 @@ if (found = ARGV.index('-u'))
 end
 
 match_credentials = lambda { |credentials|
-  got_username, got_password = credentials.to_s.unpack('m*')[0].split(':', 2)
+  got_username, got_password = credentials.to_s.unpack1('m*')[0].split(':', 2)
```

Comment:
This seems like a behaviour change. 

https://rubyapi.org/2.7/o/string#method-i-unpack1

Contrast the example - return a list and return a scalar string. 

String#[] exists, so the got_password becomes a very short string.

Reply:
Yes, thank you, my bad: I forgot to delete `[0]`. Done.